packages/framework/src/Resources/config/services/cron.yml

```diff
@@ -5,6 +5,10 @@ services:
         public: false
 
     Shopsys\FrameworkBundle\Model\Product\Elasticsearch\ProductExportCronModule:
+        tags:
+            - { name: shopsys.cron, hours: '0', minutes: '0' }
```

Comment:
What about
```suggestion
            - { name: shopsys.cron, hours: '0', minutes: '*' }
```

Reply:
That would mean it will run each five minutes if the hour is zero (= from midnight to one hour it will run this module 12 times)
Can you please explain the reason for the change? Maybe I'm missing some use-case.

Reply:
Very well then. I had in mind configuration of periodical tasks in jenkins which behaves differently. (`*` means any random value between 0 and 59 for minutes).